ocs_ci/helpers/dr_helpers.py

```
@@ -4032,6 +4032,33 @@ def validate_protection_label(kind, namespace, protection_name=None):
+    )
+    gitops_private_repo_secret["metadata"][
+        "name"
+    ] = constants.GITOPS_PRIVATE_REPO_SECRET
+
+    gitops_private_repo_secret["stringData"]["url"] = config.ENV_DATA.get(
+        "dr_workload_repo_url"
+    )
+    gitops_private_repo_secret["stringData"]["password"] = config.clusters[
+        get_active_acm_index()
+    ].AUTH["ibm_hci"]["github_token"]
+    gitops_private_repo_secret_yaml = tempfile.NamedTemporaryFile(
```

Ruff (0.15.21)

[warning] Use a context manager for opening files

(SIM115)